test/common/RcclMockFuncs.hpp

```diff
@@ -1,7 +1,15 @@
+#ifndef RCCL_MOCK_FUNCS_HPP
+#define RCCL_MOCK_FUNCS_HPP
+
+#ifndef RCCL_EXPOSE_STATIC
+
 #include "info.h"
 #include "comm.h"
 
 void ncclDebugLog(ncclDebugLogLevel, unsigned long, char const*, int, char const*, ...) {};
 ncclResult_t getHostName(char* hostname, int maxlen, const char delim) {
   return ncclSuccess;
 }
+#endif  // RCCL_EXPOSE_STATIC
```

Comment:
I think it should be the responsibility of the code including this header to exclude it if necessary. These checks are duplicated elsewhere.

```suggestion
void ncclDebugLog(ncclDebugLogLevel, unsigned long, char const*, int, char const*, ...) {};
ncclResult_t getHostName(char* hostname, int maxlen, const char delim) {
  return ncclSuccess;
}
```

Reply:
I have made modifications per your comment. `info.h` is still require in this file to get access to `ncclDebugLogLevel` parameter.